aspnet/SameSite/system-web-samesite.md

```diff
@@ -10,20 +10,59 @@ uid: samesite/system-web-samesite
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-SameSite is an [IETF](https://ietf.org/about/) draft designed to provide some protection against cross-site request forgery (CSRF) attacks. The [SameSite 2019 draft](https://tools.ietf.org/html/draft-west-cookie-incrementalism-00):
+SameSite is an [IETF](https://ietf.org/about/) draft standard designed to provide some protection against cross-site request forgery (CSRF) attacks. Originally drafted in [2016](https://tools.ietf.org/html/draft-west-first-party-cookies-07), the draft standard was recently updated in [2019](https://tools.ietf.org/html/draft-west-cookie-incrementalism-00). The updated standard is not fully compatible with the previous standard, with the following being the most noticeable differences:
 
-* Treats cookies as `SameSite=Lax` by default.
-* States cookies that explicitly assert `SameSite=None` in order to enable cross-site delivery should be marked as `Secure`.
+* Cookies without SameSite header are treated as `SameSite=Lax` by default.
+* `SameSite=None` must be used to allow cross-site cookie use.
+* Cookies that assert `SameSite=None` must also be marked as `Secure`.
```

Comment:
* The value `SameSite=None` is not allowed by the 2016 standard and causes those implementations to treat cookies as `SameSite=Strict`. See Supporting older browsers in this document.

Reply:
https://tools.ietf.org/html/draft-west-first-party-cookies-07 does mention 'SameSite=None' as a valid option in 4.2.1. Why it screws up Safari... idk. Doesn't seem like it should based on the 2016 draft standard. Maybe something like
"The value SameSite=None may cause some browsers to treat cookies as SameSite=Strict. See Supporting older browsers in this document."

Reply:
> does mention 'SameSite=None' as a valid option in 4.2.1.

4.2.1 describes internal state management, not the cookie format. See 3.1 for the cookie format: `   samesite-value = "Strict" / "Lax"`

Safari was on spec until Chrome re-wrote the spec.


https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-3.1